nodejs/dist/uws.js

```diff
@@ -25,17 +29,22 @@ class Socket {
      * @param {Function} f
      */
     on(eventName, f) {
+        if (typeof f !== 'function') {
+            throw new TypeError('Event listener must be a function');
+        }
```

Comment:
LOL. I like your choice of words. Alright, it's your baby, I'll remove it. But personally I'm a big fan of making things more debuggable especially when this is not in a hot code path.